swagger/control/swagger.yaml

```diff
@@ -0,0 +1,152 @@
+swagger: '2.0'
+info:
+  title: Mailgun Relayery Control API
+  description: |
+    API for managing the authentication tokens and channel settings of remote instances.
+  version: "1.0.0"
+host: mailgun.relayery.control.parquery.com
+schemes:
+  - https
+basePath: /
+tags:
+- name: "control"
+paths:
+  /api/token:
+    put:
+      operationId: put_token
+      tags:
+        - control
+      description: |
+        updates the token associated with the descriptor.
+
+        If there is already a token associated with the descriptor, the old token is overwritten with the new one.
+      parameters:
+        - name: named_token
+          in: body
+          schema:
+            $ref: "#/definitions/NamedToken"
+          required: true
+      consumes:
+        - application/json
+      responses:
+        200:
+          description: signals that the token was accepted.
+        default:
+          description: contains an unexpected error.
+    delete:
+      operationId: delete_token
+      tags:
+        - control
+      description: |
+        removes the token associated with the descriptor.
+      parameters:
+        - name: descriptor
+          in: body
+          schema:
+            $ref: "#/definitions/Descriptor"
+          required: true
+      consumes:
+        - application/json
+      responses:
+        200:
+          description: signals that the token was correctly erased.
+        404:
+          description: signals that no token associated to the given descriptor exists.
+        default:
+          description: contains an unexpected error.
+
+  /api/channel:
+    put:
+      operationId: put_channel
+      tags:
+        - control
+      description: |
+        updates the channel data associated with the descriptor.
```

Comment:
updates the channel uniquely identified by a descriptor.

If there is already ...